inference-engine/tests/functional/plugin/cpu/bfloat16/memory_conv.cpp

```diff
@@ -0,0 +1,203 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include <fstream>
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "common_test_utils/xml_net_builder/ir_net.hpp"
+#include "ie_system_conf.h"
+
+namespace LayerTestsDefinitions {
+using namespace CommonTestUtils;
+using namespace InferenceEngine;
+
+class MemoryConv : public testing::WithParamInterface<LayerTestsUtils::basicParams>,
+                   public LayerTestsUtils::LayerTestsCommon {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<LayerTestsUtils::basicParams> obj) {
+        InferenceEngine::Precision netPrecision;
+        InferenceEngine::SizeVector inputShapes, newInputShapes;
+        std::string targetDevice;
+        std::tie(netPrecision, inputShapes, targetDevice) = obj.param;
+
+        std::ostringstream result;
+        result << "IS=" << CommonTestUtils::vec2str(inputShapes) << "_";
+        result << "netPRC=" << netPrecision.name() << "_";
+        result << "targetDevice=" << targetDevice;
+        return result.str();
+    }
+
+protected:
+    void SetUp() {
+        SizeVector shape;
+        std::tie(inPrc, shape, targetDevice) = this->GetParam();
+
+        net_bin_v6 = make_plain_blob(Precision::FP32, SizeVector{(200 * 200 + 200 + 200 * 200 + 200) * sizeof(float)});
+        net_bin_v6->allocate();
+    }
+    std::string net_xml_v6 = R"V0G0N(
```

Comment:
Could we use Ngraph instead of hardcoded string?

Reply:
the same answer as for previous question "why do we need IR v7".